packages/@aws-cdk/aws-appsync/lib/graphqlapi.ts

```diff
@@ -300,31 +297,76 @@ export class IamResource {
   }
 }
 
+/**
+ * Attributes for GraphQL imports
+ */
+export interface GraphQLApiAttributes {
+  /**
+   * the api id of the GraphQL Api
+   */
+  readonly graphqlApiId: string,
+
+  /**
+   * the arn for the GraphQL Api
+   * @default - autogenerated arn
+   */
+  readonly graphqlArn?: string,
+}
+
 /**
  * An AppSync GraphQL API
+ *
+ * @resource AWS::AppSync::GraphQLApi
  */
-export class GraphQLApi extends Construct {
+export class GraphQLApi extends GraphQLApiBase {
+  /**
+   * Import a GraphQL API through this function
+   * @param scope scope
+   * @param id id
+   * @param attrs GraphQL API Attributes of an API
+   */
+  public static fromGraphQLApiAttributes(scope: Construct, id: string, attrs: GraphQLApiAttributes): IGraphQLApi {
```

Comment:
same comment re: casing - although we don't need to clean every scenario up, we should not proliferate something we know won't be idiomatic in Python. let's go for what we want to end up with at the outset instead of creating future chore work.